httpapi/routes.go

```diff
@@ -0,0 +1,20 @@
+package httpapi
+
+import (
+	"github.com/nibble-4bits/ondemand-go-bootcamp/usecase"
+
+	"github.com/gin-gonic/gin"
+)
+
+func getPokemonByID(r *gin.Engine, service usecase.PokemonService) {
+	r.GET("/pokemon/:id", getPokemonByIDController(service))
+}
+
+func getAllPokemons(r *gin.Engine, service usecase.PokemonService) {
+	r.GET("/pokemon", getAllPokemonsController(service))
```

Comment:
According to best practices about REST APIs, just go with plural nouns. It makes it easier for humans to understand the meaning of a collection without actually checking it.

It sounds better `GET pokemons` (in plural) instead of `GET pokemon`.

Same for the previous endpoint.
```suggestion
	r.GET("/pokemons", getAllPokemonsController(service))
```

Also, it would be a good idea to version your API; maybe in the future you develop further versions of your API, and better be ready early for documentation purposes, and avoid big code refactors.